.github/workflows/development.yml

```
@@ -34,7 +31,7 @@ jobs:
             ruby: head
             experimental: true
           - os: ubuntu
-            ruby: 2.6
+            ruby: 3.0
```

```suggestion
            ruby: "3.0"
```

Due to YAML "float to string" serialization, this bare 3.0 would become "3", which may not be what we want.

---

Haha great.